README.md

```diff
@@ -7,6 +7,56 @@ gRPC-Java works with JDK 6. TLS usage typically requires using Java 8, or Play
 Services Dynamic Security Provider on Android. Please see the [Auth
 Readme](AUTH-README.md).
 
+Download
+--------
+
+Download [the JAR][]. Or for Maven, add to your `pom.xml`:
+```
+<dependency>
+  <groupId>io.grpc</groupId>
+  <artifactId>grpc-all</artifactId>
+  <version>0.7.0</version>
+</dependency>
+```
+
+Or for Gradle, add to your dependencies:
+```
+compile 'io.grpc:grpc-all:0.7.0'
+```
+
+[the JAR]: https://search.maven.org/remote_content?g=io.grpc&a=grpc-all&v=0.7.0
+
+Development snapshots are available in [Sonatypes's snapshot
+repository](https://oss.sonatype.org/content/repositories/snapshots/).
+
+For protobuf-based codegen, use [protobuf-gradle-plugin][]:
```

Comment:
We probably want to add a link to http://sergei-ivanov.github.io/maven-protoc-plugin/examples/protoc-plugin.html for Maven users.